### DISCUSSION TITLE
Lsp documentation (hover) to go to a split.

### DISCUSSION THREAD

**Discussion by vishfrnds** [neovim/neovim#13855] [General]
Usage, how to copy text form documentation. Often documentation is big and have example of usage. Current lsp setup only supports hover for documentation, form which text can not be copied.
Most probably this should be doable in config file also, so if someone has already done it will really help.

**Comment by zeertzjq** [comment]
Calling `vim.lsp.buf.hover()` again when the hover is already shown allowing focusing the hover.
